### PR TITLE
chore: convert from npm to yarn

### LINKED ISSUE #291: build: Convert the project to use yarn
On the last [UG meeting](https://wiki.hyperledger.org/display/ARIES/2022-04-12+Aries+Bifold+Users+Group+Community+Meeting) we discussed converting the project to user Yarn. This was widely supported so this ticket to track moving ahead with the change.

### PR DESCRIPTION
**Pull request by cvarjao**
Signed-off-by: Clécio Varjão <1348549+cvarjao@users.noreply.github.com>

# Summary of Changes

Convert the project dependency manager from npm to yarn.

- [x] Test on Android
- [x] Test on iPhone
- [x] Delete `package-lock.json`, `core/package-lock.json`, `app/package-lock.json`

# Related Issues

Fixes #291

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

**Comment by jleach** [comment]
Will need to add momentJS because it was just added as part of PR #603.

**Comment by cvarjao** [comment]
@jleach , I've updated to include latest changes to dependencies. I've done as much as test as I could, I really need other people feedback if they have any issue with this setup. I am switching to "Ready for Review", and maybe add this to agenda for the next user group meeting

**Comment by jleach** [comment]
Tested from a fresh clone. Works great.

**Review by jleach — DISMISSED** [review]
Tested from a fresh clone. Works great. Zero issues. LGTM.

**Review by amanji — APPROVED** [review]

File: DEVELOPER.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Developers Guide
+
+* [Prerequisite software](#prerequisite-software)
+
+## Prerequisite software
+
+Before you can proceed with building and testing the BC Wallet app, you must install and configure the
```
Comment (JamesKEbert) [inline DEVELOPER.md:7]: Should be fixed

**Pull request merged**